src/scripts/update-icons

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
This file contains bashisms, so better request bash explicitly.

```suggestion
#! /bin/bash
```

Reply:
Where does it do so? It runs perfectly fine with dash, plus both shellcheck and checkbashisms do not say anything.

Reply:
Seems like I was wrong, I updated shellcheck. In any case, I think using bash makes sense, as it allows you to `set -euo pipefail`.